Common/Source/Draw/LKProcess.cpp

```diff
@@ -870,6 +870,15 @@ bool MapWindow::LKFormatValue(const short lkindex, const bool lktitle, TCHAR *Bu
 			_stprintf(BufferUnit, TEXT("%s"),(Units::GetHorizontalSpeedName()));
 			_stprintf(BufferTitle, TEXT("%s"), Data_Options[lkindex].Title );
 			break;
+
```

Comment:
At the beginning I wrote:
Keep this list sorted, so that the compiler can use a jump (branch) table ..
so the best place to place item 133 is after 132, not after 42.. no?